mne/io/bti/bti.py

```diff
@@ -1072,7 +1085,7 @@ def _get_bti_info(pdf_fname, config_fname, head_shape_fname, rotation_x,
                   translation, convert, ecg_ch, eog_ch, rename_channels=True,
                   sort_by_ch_name=True):
 
-    if not isinstance(pdf_fname, six.BytesIO):
+    if not isinstance(pdf_fname, six.BytesIO) and pdf_fname is not None:
```

Comment:
I'd put the `pdf_fname is not None` check first, it saves one comparison if it is `None`